eng/common/spelling/Invoke-Cspell.ps1

```diff
@@ -170,10 +170,10 @@ try {
   npm ci | Write-Host
 
   # Use the mutated configuration file when calling cspell
-  $command = "npx cspell $JobType --config $CSpellConfigPath --no-must-find-files --root $SpellCheckRoot --relative"
+  $command = "npx --no cspell $JobType --config $CSpellConfigPath --no-must-find-files --root $SpellCheckRoot --relative"
```

Comment:
Who names a command line flag `--no` ?! There's really no alternative?

Reply:
@mikeharder isn't there a better command to use in ci than `npx`?  maybe `npm exec`?

Reply:
The safest command to run a pre-installed package is:

```
npm exec --no -- <command> <args>
```

Examples:

https://github.com/search?q=repo%3AAzure%2Fazure-rest-api-specs%20%22npm%20exec%22&type=code

Reply:
The `--yes`/`--no` options are documented here:

https://docs.npmjs.com/cli/v7/commands/npm-exec#description